osu.Game/Screens/Multi/Screens/MultiplayerScreen.cs

```diff
@@ -15,6 +15,11 @@ public abstract class MultiplayerScreen : OsuScreen
 
         protected virtual Container<Drawable> TransitionContent => Content;
 
+        /// <summary>
+        /// The type to display in the title of the <see cref="Header"/>.
+        /// </summary>
+        public abstract string Type { get; }
```

Comment:
`Type => Title` for a sane default? then you don't need to add anything to `Lounge`.

Reply:
In neither case does `Type == Title`, in lounge it's `lounge` for type and `Lounge` for title.